src/main/java/com/syncstream_dev/syncstream/model/Genre.java

```diff
@@ -0,0 +1,11 @@
+package com.syncstream_dev.syncstream.model;
+
+public enum Genre {
```

Comment:
Reminder: "Dynamically" load genres into `Genre`. (We might have to use a `class` instead in the future)

Reply:
Agreed, made a draft in the backlog for this.